src/pages/TabDetails.tsx

```diff
@@ -76,7 +81,17 @@ export function useInfoForOneCycle(idx: number): InfoOneCycle {
   const dateStart: Date = new Date(cycles[idx].startDate);
   const dateFinish: Date = new Date(cycles[idx].startDate);
   dateFinish.setDate(dateFinish.getDate() + cycleLenNumber - 1);
-  const dates = `${dateStart.toLocaleDateString()} - ${dateFinish.toLocaleDateString()}`;
+  let dates = `${format(dateStart, "MMM dd")} - ${format(
+    dateFinish,
+    "MMM dd",
+  )}`;
+  if (t("locale") === "ru") {
```

Comment:
What will be here if we'll introduce a new language?
We can move this format generation inside a some function where we have `Map` or `Object` with locale locale and date-dns's locale as value

Reply:
Yes, I move this in a separate ```getFormattedDate``` function in the ```CalculationLogic``` file. I also wrote tests.